lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -219,6 +219,18 @@ extension DomElementExtension on DomElement {
   external void remove();
   external void setAttribute(String name, Object value);
   void appendText(String text) => append(createDomText(text));
+  external void removeAttribute(String name);
+  external set tabIndex(int? value);
+  external int? get tabIndex;
+  external void focus();
+  int get scrollTop =>
+      js_util.getProperty(this, 'scrollTop').round();
+  set scrollTop(int value) =>
+      js_util.setProperty<int>(this, 'scrollTop', value.round());
```

Comment:
I don't think you need to call `round` here, it's already an `int`, right?

Reply:
I'm not sure it is guaranteed to be an int. Per the mdn: `Warning: On systems using display scaling, scrollTop may give you a decimal value.` For safety, I'm going to leave the round in here and below.

Reply:
Per offline discussion, I erroneously thought the comment referred to the `getter`. The `setter` case is more complicated. I strongly suspect @eyebrowsoffire is right, and the `.round()` is mostly spurious. The only issue is if the `int` in question flowed from a pre-static interop JS API. It might still be okay, but while we're migrating it doesn't hurt to leave the round in place.

I added a comment to this effect, and we should revisit after the migration is complete.